nxdrive/objects.py

```diff
@@ -170,6 +170,7 @@ def from_dict(doc: Dict[str, Any], parent_uid: str = None) -> "NuxeoDocumentInfo
             uid = doc["uid"]
             path = doc["path"]
             props = doc["properties"]
+            file_content_data = doc["properties"].get("file:content").get("data")
```

Comment:
We can end on such error:
```python
AttributeError: 'NoneType' object has no attribute 'get'
```
To prevent this, you could use something like (also renamed the var for clarity and reusing the previsous `props` to save one dict lookup):
```python
has_data = props.get("file:content", {}).get("data")
```